docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc

```diff
@@ -46,4 +46,11 @@ After upgrading the build pipeline, you can also customize it:
 . Find the new `.tekton` directory, which is the result of upgrading the build pipeline. 
 . Modify the contents of this directory to change how {ProductName} builds the component. 
 
+.Security
+To reinforce the security of your custom build pipeline, you can follow these 3 steps:
 
+* Add an `OWNERS` file to the `.tekton` directory and list trusted contributors there. To learn more, see Kubernetes docs about link:https://www.kubernetes.dev/docs/guide/owners/[OWNERS files].
```

Comment:
I think the links in the middle of the procedure will encourage users to leave the procedure before completing it. I'll leave a comment at the end of my review, including the links in an "Additional resources" section, and you can see if you like that flow better. 

So, I'd recommend this step simply being:

1. Add an `OWNERS` file to the `.tekton` directory and list trusted contributors there.

Reply:
I don't mind doing it either way. My only concern is that "Additional resources" at the end will refer to the whole guide (Upgrading the build pipeline) and not to this Security section only.

This new content was meant to become a part of [Upgrading the build pipeline](https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/how-to-guides/proc_upgrade_build_pipeline/) Christian mentioned he'd restructure this guide this week so probably he'll find a nice way of handling this content and the links?